DesignSafe-CI/portal
# Task/des 2713 job history modal  adjustments

## Overview: ##
- URL with job uuid opens job modal
- Job modal buttons work

## PR Status: ##


## Related Jira tickets: ##

* [DES-2713](https://tacc-main.atlassian.net/browse/DES-2713)

## Testing Steps: ##
1. Click "View Details" on a job and confirm it opens modal and changes url
2. Refresh page with job uuid in url and confirm it opens modal
3. Click buttons and make sure they work

## UI Photos:
<img width="1040" alt="Screenshot 2024-05-23 at 1 18 14 PM" src="https://github.com/DesignSafe-CI/portal/assets/20326896/7dda33ee-fe72-424c-ae85-b7b2afc309c5">


## Notes: ##
